src/basic/strv.h

```diff
@@ -77,7 +77,7 @@ int strv_split_full(char ***t, const char *s, const char *separators, ExtractFla
 static inline char **strv_split(const char *s, const char *separators) {
         char **ret;
 
-        if (strv_split_full(&ret, s, separators, 0) < 0)
+        if (strv_split_full(&ret, s, separators, EXTRACT_RETAIN_ESCAPE) < 0)
```

Comment:
Hmm, indeed, this is interesting. If this is done, escaping of the separator cannot be done. I think *some* users of strv_split() expect it, while others not. It's something to investigate independently.

Reply:
add a flags parameter maybe?

Reply:
Depends on how the prior `strv_split` with `strv_split_extract` behaved before https://github.com/systemd/systemd/commit/0645b83a40d1c782f173c4d8440ab2fc82a75006 I suppose.